tests/unittest/llmapi/apps/utils.py

```
@@ -173,3 +178,103 @@
+    FILL_CHAR = b'\0'
+    SIOCGIFCONF = 0x8912
+    sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+    names = array.array('B', MAX_BYTES * FILL_CHAR)
+    names_address, names_length = names.buffer_info()
+    mutable_byte_buffer = struct.pack('iL', MAX_BYTES, names_address)
+    mutated_byte_buffer = fcntl.ioctl(sock.fileno(), SIOCGIFCONF,
+                                      mutable_byte_buffer)
+    max_bytes_out, names_address_out = struct.unpack('iL', mutated_byte_buffer)
+    namestr = names.tobytes()
+    namestr[:max_bytes_out]
+    bytes_out = namestr[:max_bytes_out]
```

Ruff (0.12.2)

Local variable `bytes_out` is assigned to but never used

Remove assignment to unused variable `bytes_out`

(F841)